cernopendata/modules/fixtures/data/docs/delphi-guide-docker/delphi-guide-docker.md

```diff
@@ -1,80 +1,72 @@
 ## Running DELPHI software from a container
 An alternative way to run DELPHI software is to use a container image.
-The following guide assumes that you are using [Podman](https://podman.io/), nevertheless it is expected to work in a similar way with other container management technologies, such as Docker.
-The images are automatically created from a CI when the software stack is being build, and contain the full software stack.
+The following guide assumes that you are using [Podman](https://podman.io/), nevertheless it is expected to work in a similar way with other container management technologies, such as [Docker](https://www.docker.com/).
+The required images to run the DELPHI software are automatically created from a CI when the software stack is being build, and contain the full software stack.
 
 ## Available images
-Images are available in the registry of the deployment project at CERNs gitlab instance, https://gitlab.cern.ch/delphi/deployment/container_registry
+Images are available in the registry of the `DELPHI deployment` project at [CERNs gitlab instance](https://gitlab.cern.ch/delphi/deployment/container_registry).
```

Comment:
The link name is to the DELPHI repository, but the link title says it's CERN gitlab instance generically, which may be confusing.

I would just make "DELPHI deployment" to be the link and the rest of the phrase would be free text, such as:

```
Images are available in the registry of the [DELPHI deployment project](https://gitlab.cern.ch/delphi/deployment/container_registry) at CERN GitLab.
```